### ISSUE TITLE
Option to use primary IP for 2 hosts, and proxies for the rest of them

### ISSUE THREAD

**Issue by iwishiknewcoding** [gabrielbrop/haxball-server#13]
With the current implementation, is it possible to use up the 2 primary slots of non-proxy IP for hosts, and then start assigning proxies to rest of the hosts? Because as far as I'm seeing, `proxyEnabled` makes every host to go through the proxy, and the primary 2 slots are getting lost.

**Comment by gabrielbrop** [comment]
No, this is not possible. You can add your primary IP to the proxy server (and then to the beginning of the `proxyServers` array) to avoid wasting available IPs.
